.github/styles/Google/Acronyms.yml

```diff
@@ -1,7 +1,7 @@
 extends: conditional
 message: "Spell out '%s', if it's unfamiliar to the audience."
 link: 'https://developers.google.com/style/abbreviations'
-level: warning
+level: error
```

Comment:
side question: I am not sure if these `.github/styles` file have to live in `.github` directory or not. can they be put in a non `.github` directory? like in `.vale` or something?

Reply:
They don't have to live in `.github`. I followed the layout we use for the Anyscale docs: https://github.com/anyscale/docs/tree/master/.github/styles.

I've moved the files to `.vale`.